src/utils/sortPeople.ts

```diff
@@ -0,0 +1,37 @@
+import { Person } from '../types/Person';
+
+export const sortPeople = (
+  people: Person[],
+  sortType: string | null,
```

Comment:
Consider replacing 'string | null' with a more specific type or enum for sortType parameter to avoid passing incorrect values.